swig/python2/Makefile.am

```diff
@@ -3,14 +3,15 @@ AM_CFLAGS = -fPIC
 PYLIBVER ?= python$(PYTHON_VERSION)
 
 AM_CPPFLAGS =	@xml2_CFLAGS@ \
+		-I${top_srcdir}/src/common/public \
 		-I$(top_srcdir)/src/CPE/public \
+		-I$(top_srcdir)/src/CVE/public \
 		-I$(top_srcdir)/src/CVSS/public \
+		-I$(top_srcdir)/src/DS/public \
 		-I$(top_srcdir)/src/OVAL/public \
+		-I${top_srcdir}/src/source/public \
 		-I$(top_srcdir)/src/XCCDF/public \
 		-I$(top_srcdir)/src/XCCDF_POLICY/public \
-		-I${top_srcdir}/src/common/public \
-		-I${top_srcdir}/src/source/public \
-		-I$(top_srcdir)/src/CVE/public \
```

Comment:
Ah, understood. The reason why I frown upon changes like this is because this is in multiple branches and will cause conflicts as we forward-port fixes.